packages/website/components/account/filesManager/filesManager.js

```diff
@@ -345,6 +347,16 @@ const FilesManager = ({ className, content, onFileUpload }) => {
           onChange={setSortedFiles}
           onSelectChange={showCheckOverlayHandler}
         />
+        <Dropdown
+          className="files-manager-gateway"
+          staticLabel="Gateway"
+          value={linkPrefix}
+          options={[
+            { value: 'https://w3s.link/ipfs/', label: 'w3s' },
```

Comment:
```suggestion
            { value: 'https://w3s.link/ipfs/', label: 'w3link' },
```

Probably better to name the product name instead